File: code/c-sharp/x64/src/OopSolidSample/Models/Home/HomeIndexViewModel.cs
using System.Collections.Generic;

namespace OopSolidSample.Models.Home
{
    public class HomeIndexViewModel
    {
        public class GameViewModel
        {
            public string Name { get; set; }
            public string Link { get; set; }
            public string HighScoreName { get; set; }
            public int? HighScore { get; set; }
        }

        public IList<GameViewModel> Games { get; set; }

        public HomeIndexViewModel()
        {
            Games = new List<GameViewModel>();
        }
    }
}

File: code/java/src/oopSolidSample/models/home/HomeIndexViewModel.java
package oopSolidSample.models.home;

import java.util.ArrayList;
import java.util.List;

public class HomeIndexViewModel
{
    public class GameViewModel
    {
        public String Name;
        public String Link;
        public String HighScoreName;
        public int HighScore;
    }

    public List<GameViewModel> Games;

    public HomeIndexViewModel()
    {
        Games = new ArrayList<GameViewModel>();
    }
}